content/ko/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -0,0 +1,225 @@
+---
+title: 레이블과 셀렉터
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+_레이블_은 파드와 같은 오브젝트에 첨부된 키와 값의 쌍이다.
```

Comment:
Please add a space like `_레이블_ 은` because of wrong rendering on https://deploy-preview-15831--k8s-dev-ko.netlify.com/ko/docs/concepts/overview/working-with-objects/labels/ like the screenshot.
<img width="404" alt="image" src="https://user-images.githubusercontent.com/6965982/63320010-36c59400-c357-11e9-93f0-f151aa46d07a.png">

Reply:
아차.. 띄어쓰기 해주시며 되겠네요.
```suggestion
_레이블_ 은 파드와 같은 오브젝트에 첨부된 키와 값의 쌍이다.
```